compiler/src/org.graalvm.compiler.core.llvm/src/org/graalvm/compiler/core/llvm/LLVMIRBuilder.java

```diff
@@ -77,7 +77,7 @@ protected LLVMIRBuilder(String functionName, LLVMContextRef context, boolean tra
              * doesn't recognize an address space cast (see pointerType()) as declaring a new
              * reference, but it does a function return value.
              */
-            gcRegisterFunction = addFunction("__svm_gc_register", functionType(objectType(), rawPointerType()));
+            gcRegisterFunction = addFunction(LLVMUtils.GC_REGISTER_FUNCTION_NAME, functionType(objectType(), rawPointerType()));
             LLVM.LLVMSetLinkage(gcRegisterFunction, LLVM.LLVMLinkOnceAnyLinkage);
             setAttribute(gcRegisterFunction, LLVM.LLVMAttributeFunctionIndex, "alwaysinline");
             setAttribute(gcRegisterFunction, LLVM.LLVMAttributeFunctionIndex, "gc-leaf-function");
```

Comment:
Can we use constant strings for these? I've seen `gc-leaf-function` a few times during this review.

Reply:
Fixed.